plugin/src/test/java/io/jenkins/plugins/analysis/core/scm/GitBlamerITest.java

```diff
@@ -1,438 +1,400 @@
-        assertSuccessfulBlame(result, 11, 3);
-
-        TablePageObject table = getBlamesTable(result);
-        assertElevenIssues(commits, table);
-
-        table.filter("LoremIpsum.java");
-        assertThat(table.getInfo()).isEqualTo("Showing 1 to 4 of 4 entries (filtered from 11 total entries)");
-        List<TableRowPageObject> rows = table.getRows();
-        assertThat(rows).hasSize(4);
-        assertColumnsOfRowLoremIpsum(rows.get(0), 1, commits.get("LoremIpsum"));
-        assertColumnsOfRowLoremIpsum(rows.get(1), 2, commits.get("LoremIpsum"));
-        assertColumnsOfRowLoremIpsum(rows.get(2), 3, commits.get("LoremIpsum"));
-        assertColumnsOfRowLoremIpsum(rows.get(3), 4, commits.get("LoremIpsum"));
-    }
-
-    /**
-     * Test if blaming works on a build out of tree. See JENKINS-57260.
-     *
-     * @throws Exception
-     *         if there is a problem with the git repository
-     */
-    @Test @Issue("JENKINS-57260")
-    public void shouldBlameWithBuildOutOfTree() throws Exception {
-        gitRepo.init();
-        createAndCommitFile("Test.h", "#ifdef \"");
-
-        String firstCommit = gitRepo.head();
-
-        createAndCommitFile("Jenkinsfile", "pipeline {\n"
-                + "  agent any\n"
-                + "  options {\n"
-                + "    skipDefaultCheckout()\n"
-                + "  }\n"
-                + "  stages {\n"
-                + "    stage('Prepare') {\n"
-                + "      steps {\n"
-                + "        dir('source') {\n"
-                + "          checkout scm\n"
-                + "        }\n"
-                + "      }\n"
-                + "    }\n"
-                + "    stage('Doxygen') {\n"
-                + "      steps {\n"
-                + "        dir('build/doxygen') {\n"
-                + "          echo 'Test.h:1: Error: Unexpected character'\n"
-                + "        }\n"
-                + "        recordIssues(aggregatingResults: true, "
-                + "             enabledForFailure: true, "
-                + "             tool: doxygen(name: 'Doxygen'), "
-                + "             sourceDirectory: 'source'"
-                + "        )\n"
-                + "      }\n"
-                + "    }\n"
-                + "  }\n"
-                + "}");
-
-        WorkflowJob project = createPipeline();
-        project.setDefinition(new CpsScmFlowDefinition(new GitSCM(gitRepo.toString()), "Jenkinsfile"));
-
-        AnalysisResult result = scheduleSuccessfulBuild(project);
-        assertSuccessfulBlame(result, 1, 1);
-
-        TablePageObject table = getBlamesTable(result);
-        assertThat(table.getInfo()).isEqualTo("Showing 1 to 1 of 1 entries");
-        assertThat(table.size()).isEqualTo(1);
-        
-        List<TableRowPageObject> rows = table.getRows();
-        assertThat(rows).hasSize(1);
-        assertThat(rows.get(0).getValuesByColumnLabel()).contains(
```

Comment:
Da wäre noch die Idee gewesen, die Table zu prüfen, analog zu meinem Kommentar  in https://github.com/jenkinsci/warnings-ng-plugin/pull/466#discussion_r426424677

Reply:
Hallo @uhafner,

ich habe angefangen Ihren Kommentar einzuarbeiten und das TableModel zu überprüfen, allerdings habe ich jetzt das Problem, dass wenn ich die Methode "getIssuesModel" mit dem TableModel ("blames") aufrufe, z.B. in der Methode "shouldBlameElevenIssuesWithFreestyle()", dann bekomme ich 11 mal dasselbe Blame-Object zurück ... 

Wissen Sie woran das liegen könnte?

Reply:
Können Sie mal mit dem Master morgen, da scheint noch ein Problem vorher aufzutreten...

Reply:
Also bei mir kommen da 11 verschiedene Zurück. Wenn ich den Jenkins beim Breakpoint öffne, kommt auch: 

![Bildschirmfoto 2020-06-02 um 20 28 53](https://user-images.githubusercontent.com/503338/83556181-be0f7d00-a50f-11ea-9e72-8aad3e0d5e4a.png)

Reply:
Ihre Assertions passen noch nicht: `assertThat(object).isEqualTo("bla");`

Reply:
Hallo Herr Professor @uhafner,

entschuldigen Sie bitte, dass es jetzt doch etwas länger gedauert hat, aber ich musste das Projekt nochmal neu aufbauen, irgendwo war da ein krummer Hund ... am Ende ging dann alles und ich hoffe, dass die Änderung so i.O. sind.